core/server/common/src/main/java/alluxio/master/StateLockManager.java

```diff
@@ -123,6 +124,9 @@ public void mastersStartedCallback() {
    * @throws InterruptedException
    */
   public LockResource lockShared() throws InterruptedException {
+    if (LOG.isDebugEnabled()) {
+      LOG.debug("Thread-%d entered lockShared().", Thread.currentThread().getId());
```

Comment:
do you also want thread name, or just the id? Also, I think for logging you need "{}", not "%d". "%d" I think is for `String.format()`.